Remotion/Security/Core/Metadata/StatePropertyReflector.cs

```diff
@@ -58,25 +58,25 @@ public StatePropertyInfo GetMetadata (PropertyInfo property, MetadataCache cache
       if (!property.PropertyType.IsEnum)
       {
         throw new ArgumentException (
-            string.Format ("The type of the property '{0}' in type '{1}' is not an enumerated type.", property.Name, property.DeclaringType.GetFullNameSafe()),
+            string.Format ("The type of the property '{0}' in type '{1}' is not an enumerated type.", property.Name, property.DeclaringType!.GetFullNameSafe()),
```

Comment:
Did we use null-bang or null-propagation for exceptions in earlier PRs? I know we use null-propagation for logging and when rethrowing an exception, but what about the message of initial exceptions?

Reply:
Null bangs were used